install/installer/pkg/components/workspace/ide/constants.go

```diff
@@ -10,7 +10,6 @@ const (
 	Code1_85IDEImageStableVersion = "commit-cb1173f2a457633550a7fdc89af86d8d4da51876"
 	CodeHelperIDEImage            = "ide/code-codehelper"
 	CodeWebExtensionImage         = "ide/gitpod-code-web"
-	CodeWebExtensionVersion       = "commit-4e069a6195f3926ba8b84725bc806228f4cb94ec" // gitpod-web extension version comes from https://github.com/gitpod-io/gitpod-code
```

Comment:
Should `CodeIDEImageStableVersion` also be moved to `WORKSPACE.yaml`? Maybe in a subsequent PR?

Reply:
Aye, sorry, I was asking for posterity. I agree this is unrelated/non-blocking.

Reply:
> This may require a redesign.

For a later PR, of course?